### ISSUE TITLE
Default  Grafana docker name is agraf-3000

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla-monitoring#211]
```
>./start-all.sh
>sudo docker ps
CONTAINER ID        IMAGE                    COMMAND                  CREATED             STATUS              PORTS                                                    NAMES
dff03413dfcf        grafana/grafana:4.1.1    "/run.sh"                16 seconds ago      Up 14 seconds       0.0.0.0:3000->3000/tcp                                   agraf-3000
113c8b8adc88        prom/prometheus:v1.5.2   "/bin/prometheus -..."   21 seconds ago      Up 20 seconds       0.0.0.0:9090->9090/tcp                                   aprom
```
Before  https://github.com/scylladb/scylla-grafana-monitoring/pull/210
the Grafana docker name was just agraf. This make kill-all.sh script incompatible

**Comment by tzach** [comment]
@duarten please take a look, I think this is a result of #210 which I already merged.
Please send a new PR with a fix.
Thanks

**Comment by duarten** [comment]
I suck at scripting.

**Comment by duarten** [comment]
Fixed by ffaa45a7e89d0e1eb3832ab20eb945011108c9c4.
